src/main/java/io/jenkins/plugins/luxair/ImageTag.java

```diff
@@ -107,10 +108,9 @@ private static String getAuthToken(String[] authService, String image, String us
             logger.info("HTTP status: " + response.getStatusText());
             response.getBody().getObject()
                 .getJSONArray("tags")
-                .forEach(item -> tags.add(item.toString()));
+                .forEach(item -> tags.add(new VersionNumber(item.toString())));
         } else {
             logger.warning("HTTP status: " + response.getStatusText());
-            tags.add(" " + response.getStatusText() + " !");
```

Comment:
Sidenote: 
This error message practically would never display, unless the filter was left as the default regex `.*`.

To still display a configuration error (aka. no tags in selection) we could do some logic in the .jelly file or simply keep the empty select.
Let me know if that sounds like a acceptable solution to you and I will get right to it :wink:

Reply:
We need to give some feedback in case of error (mainly connection and authentication) directly to the user in some way. Leaving the empty select is not a good idea

Reply:
agreed, I'll edit the .jelly accordingly so that we still have some error msg even with a non-default filter regex

Reply:
Okay I did add back a basic version of ErrorMessage propagation, yet the current code for it is just to make it work again.
I'll open a followup refractor PR, that will aim at cleaning up this and some other code. I'll use #9 for that.

Hope this temporary solution is okay for you atm, otherwise let me know, and I'll rework the problematic bits.

Reply:
Okay since I realized that I have goofed up that "fix" of mine, since I somehow forgot that this is a STATIC context, I'll push another commit later once I have a proper solution. -.-

Reply:
Okay, I applied a rudimentary fix so that the HTTP response gets shown to the user in case of an error. [`58d2133`](https://github.com/jenkinsci/image-tag-parameter-plugin/pull/7/commits/58d21336228ed863ba0041af25d2c1407f603efd)
I will expand on this fix with a cleanup and refactoring PR in the future, should this PR get merged.

I hope this is in line with what you were thinking of :D